Rakefile

```diff
@@ -165,7 +165,17 @@ def _flatten_project
 
     puts "Working on #{Dir[dir + '/*.tf']}" if debug
     system("terraform get #{dir}")
-    FileUtils.cp( Dir["#{dir}/*.tf"], tmp_dir)
+    FileUtils.cp(Dir["#{dir}/*.tf"], tmp_dir)
+
+    # this has a small chance of overwriting a file if both resources/ and resources/$env
+    # have the same file
+    ['files', 'templates'].each do |subdir|
+      if File.directory? "#{dir}/#{subdir}/"
```

Comment:
`dir` here is `resources` or `resources/<env>` so this is working with an expected structure where the `templates` and `files` are inside `resources` rather than in <project>. Is that correct as it doesn't match up with @surminus's comment on https://github.com/alphagov/govuk-terraform-provisioning/pull/46?